LBHackney-IT/lbh-social-care
# Initial setup

# Done
- Created a basic nextjs application
- Basic Hackney branded Layout, including [GOVUK frontend lib](https://github.com/alphagov/govuk-frontend)
- Added linting (`eslint`), testing (`jest`), auto-formatting (`husky`/`prettier`)

<img width="989" alt="Screenshot 2020-10-20 at 13 48 31" src="https://user-images.githubusercontent.com/435399/96588004-6073a580-12e3-11eb-897e-da8458ce9ca4.png">


## How to test it
- checkout this branch
- install dependencies
- run dev environment `yarn dev`
- check that everything looks good :)

## Next steps
- Added some basic components
